### PR TITLE
feat(avatar): add `loading` and `fetchpriority` to avatar image

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

File: src/avatar/avatar.component.ts [hunk 1]
```diff
@@ -16,6 +16,12 @@ export const thyAvatarSizeMap = {
     lg: 48
 };
 
+/** https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-loading */
+type ThyAvatarLoading = 'eager' | 'lazy';
+
+/** https://wicg.github.io/priority-hints/#idl-index */
+type ThyAvatarFetchPriority = 'high' | 'low' | 'auto';
```
Comment (why520crazy) [inline src/avatar/avatar.component.ts:23]: It's best to export types `ThyAvatarLoading` and `ThyAvatarFetchPriority`

File: src/avatar/avatar.component.ts [hunk 2]
```diff
@@ -69,6 +75,10 @@ export class ThyAvatarComponent implements OnInit {
 
     @Input() thyDisabled: boolean;
 
+    @Input() thyLoading?: ThyAvatarLoading;
+
+    @Input() thyFetchPriority?: ThyAvatarFetchPriority;
```
Comment (why520crazy) [inline src/avatar/avatar.component.ts:80]: Add test cases for attributes loading and fetchpriority

**Review by why520crazy — CHANGES_REQUESTED** [review]

File: src/avatar/avatar.component.ts (hunk 1 above)
Comment (arturovt) [reply src/avatar/avatar.component.ts]: Done

File: src/avatar/avatar.component.ts (hunk 2 above)
Comment (arturovt) [reply src/avatar/avatar.component.ts:80]: Done

**Pull request merged**